app/components/FilterableComponents/FilterableTablePagination.tsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import {useRouter} from 'next/router';
+import {Pagination} from 'react-bootstrap';
+
+interface Props {
+  totalCount: number;
+  maxResultsPerPage: number;
+  pageInfo: {
+    hasNextPage: boolean;
+    hasPreviousPage: boolean;
+  };
+}
+export const FilterableTablePaginationComponent: React.FunctionComponent<Props> = (
+  props
+) => {
+  const {hasNextPage, hasPreviousPage} = props.pageInfo;
```

Comment:
do you need those, given that you already have the number of pages, and the active page?